otlp/src/otlp/logs/component.cpp

```diff
@@ -51,19 +64,47 @@ LoggerComponent::LoggerComponent(const components::ComponentConfig& config,
   logger_config.attributes_mapping =
       config["attributes-mapping"]
           .As<std::unordered_map<std::string, std::string>>({});
+
+  if (config.HasMember("sinks")) {
```

Comment:
no need for HasMember
no sinks will lead to defaults of As()

Reply:
if I won't add HasMember, config["sinks"] should throw, shouldn't it?

Reply:
no
only config["sinks"][smth][smth].As<T>() would throw
kMissing doesn't throw until As() is called (without default)